src/LibExpat.jl

```diff
@@ -88,8 +90,9 @@ mutable struct XPHandle
   XPHandle(p) = new(p, ETree(""), false)
 end
 
+parsed_data(xph::XPHandle) = root(xph.pdata)
 
-function xp_make_parser(sep='\0')
```

Comment:
should leave a compatible version of the existing api, or this could be breaking

Reply:
it was not exported

Reply:
unexported functions still get called occasionally, it's best not to break them if you're trying to add a new separate function. otherwise everyone needs to audit every usage of this library to check if they're using the function that you're removing, in exchange for a feature that you're the only one who is using initially

Reply:
I'm not using it. If you don't want my contribution it's not a problem. While you're arguing I've developed an entire new library to parse XML in julia as this one unusable in this state inside others libraries. This PR was a test before working harder on a re-factoring, I feel being wheel inspired to not have pushed days of works at once.